app/operator_cmds/commands.py

```diff
@@ -34,7 +35,9 @@ def run_action(client, action, operators_tuple, parallel, brew_token=None):
             kwargs["channel"] = _operator.get("channel", "stable")
             kwargs["source"] = _operator.get("source", "redhat-operators")
             kwargs["iib_index_image"] = _operator.get("iib")
-            kwargs["target_namespaces"] = _operator.get("target-namespaces")
+            kwargs["target_namespaces"] = ast.literal_eval(
+                _operator.get("target_namespaces")
+            )
```

Comment:
why is this needed?
'target_namespaces': ['ns1', 'ns2']
please refer to the `help`

Also, please do not set default. Either the user passes `target_namespaces` or it will be None